### ISSUE TITLE
Page "Not Found" error when trying to download a pdf

### ISSUE THREAD

**Issue by marlycormar** [grezniczek/redcap_pdf_actiontags#1]
Hi @grezniczek, we are experiencing an issue with `redcap_pdf_actiontags` module. When clicking any of the options under **Download PDF of instrument(s)**, the page is redirected to another page displaying the following error:

```
Not Found
The requested URL /redcap_v9.0.3/ExternalModules/ was not found on this server.
```
The page is redirecting to a url whose relative path may be incorrect:

**/redcap_v9.0.3/ExternalModules/?prefix=redcap_pdf_actiontags&amp;page=pdf&amp;pid=61&amp;render_page=my_first_instrument&amp;id=1&amp;event_id=117**

We have folks at @ctsit who can work on this issue (@ChemiKyle). Philip Chase (@pbchase) or I can review the fix.

**Comment by grezniczek** [comment]
Hi @marlycormar, this is quite peculiar. 
Does the path "/redcap_v9.0.3/ExternalModules/" (index.php) exist on your server? In other words, is the path _really_ wrong? If yes, what _should_ it look like on your server? Thanks.

**Comment by marlycormar** [comment]
Hi @grezniczek, thank you for getting back to me so quickly. 

The path `/redcap_v9.0.3/ExternalModules/` does not exist on my server per se. The correct path to get to the ExternalModules folder is `/redcap/redcap_v9.0.3/ExternalModules/` as the root of my server is at `/redcap/`. It would be nice to make the url generic enough so that it wouldn't assume that the root directory is at `/`.

On another note, if I hack the url so that it contains the root of my server, a pdf is downloaded but it is corrupted (I am unable to open the pdf).

**Comment by grezniczek** [comment]
Hi @marlycormar, thanks for the info. I could identify the problems. One was a bug that indeed caused the wrong url to be generated when REDCap was located in a subfolder relative to the server's  domain name. The other ("corrupted" pdf) apparently is caused by some change introduced with 9.0.2 or 9.0.3 that caused server responses to be gzip'ed, leading to the PDF being compressed (if you add a .zip extension to the corrupt pdfs you'll see that you can unzip them to give the proper pdf). 
Both issues have been fixed in the v1.1.1 release (available on GitHub; submitted to the EM Repo).

Thanks again for reporting and assisting in troubleshooting.
